docs/developer-guide.md

```diff
@@ -80,6 +80,7 @@ The easiest way to run these tests locally is by using commands from the Makefil
 > make pythontests              # standard pytest run
 > make pythontests_verbose      # run pytest with verbose output (helpful for troubleshooting unit tests)
 > make pythontests_watch        # adds a watcher that can run pytest in the background; unit tests will re-run whenever changes to a Python file are detected
+> make pythontests_coverage     # run pytest with coverage report
```

Comment:
I got an error when I tried to run this locally:
``` make pythontests_coverage
pytest --cov=. --cov-config=.coveragerc --cov-report=term-missing --no-cov-on-fail
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --cov=. --cov-config=.coveragerc --cov-report=term-missing --no-cov-on-fail
  inifile: /home/briannacerkiewicz/cas-registration/bc_obps/pytest.ini
  rootdir: /home/briannacerkiewicz/cas-registration/bc_obps

Reply:
I am unable to reproduce this issue. Have you executed the command `poetry install` while testing this pull request?

Reply:
Based on the conversation I had with Andrea, You should be able to fix this issue by running `poetry install`